src/screen.py

```diff
@@ -0,0 +1,128 @@
+            self.screens[1].insert(self.GRID_WIDTH-self.BOX_PADDING_X-1, screen_y, ["│"])
+
+        # Screen 2
+        for screen_x in range(0, self.GRID_WIDTH-1):
+            self.screens[2].insert(screen_x, self.GRID_HEIGHT-self.BOX_PADDING_Y-1, ["─"])
+
+        # Screen 3
+        self.screens[3].insert(self.BOX_PADDING_X - 1, self.GRID_HEIGHT-self.BOX_PADDING_Y-1, ["┐"])
+        for screen_x in range(0, self.BOX_PADDING_X-1):
+            self.screens[3].insert(screen_x, self.GRID_HEIGHT-self.BOX_PADDING_Y-1, ["─"])
+        for screen_y in range(self.GRID_HEIGHT-self.BOX_PADDING_Y, self.GRID_HEIGHT-1):
+            self.screens[3].insert(self.BOX_PADDING_X-1, screen_y, ["│"])
+
+        # Screen 4
+        for screen_y in range(0, self.GRID_HEIGHT-1):
+            self.screens[4].insert(self.GRID_WIDTH-self.BOX_PADDING_X-1, screen_y, ["│"])
+
+        # Screen 5
+        #  self.screens[5].insert(0, 0, ["┌"])
+        #  self.screens[5].insert(0, self.GRID_HEIGHT-1, ["└"])
+        #  for screen_x in range(1, self.GRID_WIDTH-1):
+        #      self.screens[5].insert(screen_x, 0, ["─"])
+        #      self.screens[5].insert(screen_x, self.GRID_HEIGHT-1, ["─"])
+        #  self.screens[5].insert(self.GRID_WIDTH-1, 0, ["┐"])
+        #  self.screens[5].insert(self.GRID_WIDTH-1, self.GRID_HEIGHT-1, ["┘"])
+        #  for screen_y in range(1, self.GRID_HEIGHT-1):
+        #      self.screens[5].insert(0, screen_y, ["│"])
+        #      self.screens[5].insert(self.GRID_WIDTH-1, screen_y, ["│"])
+
+        # Screen 6
+        for screen_y in range(0, self.GRID_HEIGHT-1):
+            self.screens[6].insert(self.BOX_PADDING_X-1, screen_y, ["│"])
+
+        # Screen 7
+        self.screens[7].insert(self.GRID_WIDTH-self.BOX_PADDING_X-1, self.BOX_PADDING_Y-1, ["└"])
+        for screen_x in range(self.GRID_WIDTH-self.BOX_PADDING_X, self.GRID_WIDTH-1):
+            self.screens[7].insert(screen_x, self.BOX_PADDING_Y-1, ["─"])
+        for screen_y in range(0, self.BOX_PADDING_Y-1):
+            self.screens[7].insert(self.GRID_WIDTH-self.BOX_PADDING_X-1, screen_y, ["│"])
+
+        # Screen 8
+        for screen_x in range(0, self.GRID_WIDTH-1):
+            self.screens[8].insert(screen_x, self.BOX_PADDING_Y-1, ["─"])
+
+        # Screen 9
+        self.screens[9].insert(self.BOX_PADDING_X-1, self.BOX_PADDING_Y-1, ["┘"])
+        for screen_x in range(0, self.BOX_PADDING_X-1):
+            self.screens[9].insert(screen_x, self.BOX_PADDING_Y-1, ["─"])
+        for screen_y in range(0, self.BOX_PADDING_Y-1):
+            self.screens[9].insert(self.BOX_PADDING_X-1, screen_y, ["│"])
+
+    def insertEntity(self, entity: Entity) -> None:
+        """Inserts a object of entity class to its specified point in the current screen.
+
+        :param person: Object, imported from entities.py
+        """
+        self.screens[self.current_screen].insert_entity(entity)
```

Comment:
Would it be beneficial to extend this to allow for an insertion on an arbitrary screen, but defaulting to the current one? Would be good in order to initialise the scenes on the different screens. What do you think?
```suggestion
    def insertEntity(self, entity: Entity, screen_number:int = self.current_screen) -> None:
        """Inserts a object of entity class to its specified point on a specified screen (defaults to current screen)

        :param person: Object, imported from entities.py
        """
        self.screens[screen_number].insert_entity(entity)
```

Reply:
Yup, that would be better.
Thought I think further changes to screen class should be continued in #18